packages/svelte/src/compiler/types/template.d.ts

```diff
@@ -93,6 +93,7 @@ export interface SvelteOptions {
 		 */
 		extend?: ArrowFunctionExpression | Identifier;
 	};
+	attributes: Array<Attribute | SpreadAttribute | Directive>;
```

Comment:
I think this could just be `Attribute[]`? Means that consumers won't have to do this to appease TypeScript:

```ts
if (attribute.type === 'Attribute') {...}
```

```suggestion
	attributes: Attribute[];
```

Reply:
Done, but I had to `@ts-ignore` the lines where attributes are assigned.